services/movie.py

```diff
@@ -1,14 +1,20 @@
-from django.db.models import QuerySet
+from django.db import transaction
+from django.db.models import QuerySet, Q
 
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
 ) -> QuerySet:
     queryset = Movie.objects.all()
 
+    if title:
+        # queryset = queryset.filter(title=title)
+        queryset = Movie.objects.filter(Q(title__contains=title))
```

Comment:
The use of `Q(title__contains=title)` is correct for partial matches, but if you want an exact match, you should use `filter(title=title)`. Ensure that the use of `Q` is intentional for partial matching.